internal/session/repository_session.go

```diff
@@ -700,6 +701,20 @@ func (r *Repository) checkIfNoLongerActive(ctx context.Context, reportedSessions
 	return notActive, nil
 }
 
+func (r *Repository) deleteTerminated(ctx context.Context, threshold time.Duration) (int, error) {
+	const op = "session.(Repository).deleteTerminated"
+
+	args := []any{
```

Comment:
Oh yeah, did we ever do the rewrite from `interface{}` -> `any`?

Reply:
No we didn't